memory_backend.go

```diff
@@ -0,0 +1,286 @@
+package neoq
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/guregu/null"
+	"github.com/iancoleman/strcase"
+	"github.com/jsuar/go-cron-descriptor/pkg/crondescriptor"
+	"github.com/pkg/errors"
```

Comment:
No need for this guy. `"errors"` is enough. This errors package used to provide errors.Wrap, which is now done with fmt.Errorf + the %w operator.